ipaclient/install/sssd.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2016  FreeIPA Contributors see COPYING for license
+#
+
+from ipalib.install import service
+from ipalib.install.service import enroll_only
+from ipapython.install.core import group, knob
+
+
+@group
+class SSSDInstallInterface(service.ServiceInstallInterface):
+    description = "SSSD"
+
+    fixed_primary = knob(
+        None,
+        description="Configure sssd to use fixed server as primary IPA server",
+    )
+    fixed_primary = enroll_only(fixed_primary)
+
+    permit = knob(
+        None,
+        description="disable access rules by default, permit all access.",
```

Comment:
This is how it was cased before, I did not (intentionally) change any of the descriptions.